### DISCUSSION TITLE
Why the answer it gives looks like this?

### DISCUSSION THREAD

**Discussion by xinyuliu006** [ggml-org/llama.cpp#8972] [Q&A]
![image](https://github.com/user-attachments/assets/d476d4d9-f682-444d-aad0-200a1f69c727)

**Comment by supportend** [comment]
Which model? Give full command line you started this is useful.

**Reply by xinyuliu006** [reply]
Here is the command I used.
```
/data/local/tmp/llama-cpp/bin/llama-cli -m /data/local/tmp/models/models-gguf/mistral-7b-instruct-v0.1.Q4_K_M.gguf --interactive --conversation --prompt "Hello, how can I help you today?"
```

**Comment by supportend** [comment]
Ok, it's a older model, i found it here: https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF

TheBloke gives this example: `-p "<s>[INST]{prompt} [/INST]"`
I download recent models here: https://huggingface.co/bartowski
